### PR TITLE
Fixing formatting issues for very small (+/-1e-23) numbers.

### PR DESCRIPTION
**Pull request by MarkHerhold**
Closes issue #145

### CODE REVIEW

File: numbro.js [hunk 1]
```diff
@@ -110,16 +135,21 @@
...
 
-        if (value.toFixed(0).search('e') > -1) {
-            // Above 1e21, toFixed returns scientific notation, which
-            // is useless and unexpected
-            output = toFixedLarge(value, precision);
+        if (value.toString().indexOf('e') > -1) {
+            // toFixed returns scientific notation for numbers above 1e21 and below 1e-7
+            output = toFixedLargeSmall(value, precision);
+            // remove the leading negative sign if it exists and should not be present (e.g. -0.00)
+            if (output.charAt(0) === '-' && Math.sign(output) === 0) {
```
Comment (MarkHerhold) [inline numbro.js]: I didn't realize Math.sign was an ES6 feature. Caused the build to fail. :(

**Comment by MarkHerhold** [comment]
@BenjaminVanRyseghem Please publish when merged. Thanks!

**Comment by MarkHerhold** [comment]
@BenjaminVanRyseghem Any issues with merging this? Thanks.

Also, I'd like to be considered being added as a contributor to help out the project.

**Comment by MarkHerhold** [comment]
@BenjaminVanRyseghem bump. Please take a look.

**Comment by BenjaminVanRyseghem** [comment]
Looks good :smile:
sorry for the delay

**Comment by BenjaminVanRyseghem** [comment]
in numbro 1.7.1

**Comment by MarkHerhold** [comment]
Thanks!

**Pull request merged**